flink/v1.16/flink/src/test/java/org/apache/iceberg/flink/source/TestFlinkMetaDataTable.java

```diff
@@ -285,7 +284,7 @@ public void testUnPartitionedTable() throws IOException {
     List<String> deleteColumns =
         deleteFilesTableSchema.columns().stream()
             .map(Types.NestedField::name)
-            .filter(c -> !c.equals(MetricsUtil.READABLE_METRICS))
+            .filter(c -> !MetadataTableUtils.DERIVED_FIELDS.contains(c))
```

Comment:
is there a way to avoid these changes in flink ?

Reply:
I think this flink test change is needed to pass build. Because we add a new data sequence number as schema change in baseFilesTable, we are excluding here.